client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectSettings.tsx

```diff
@@ -376,7 +376,7 @@ function ProjectSettingsForm({ project }: ProjectPageSettingsProps) {
               )}
             </div>
           }
-          requestedPermission="write"
+          requestedPermission="delete" // NOTE: "write" is not enough
```

Comment:
We don't need the comment, we have the same rule for changing the slug of a group, see: `client/src/features/groupsV2/settings/GroupSettingsMetadata.tsx`.

```suggestion
          requestedPermission="delete"
```